### DISCUSSION TITLE
Gitlab CI unplugging

### DISCUSSION THREAD

**Discussion by 0penBrain** [FreeCAD/FreeCAD#9535] [Maintainers discussions]
Hi @FreeCAD/maintainers,

Following our last dev meeting, CI Gitlab webhook was disabled (I guess by @chennes, not sure) on 05/07.
However I found that still newly created PRs remain in an unfinished check state, waiting for the Gitlab CI to report its results.

I found the Gitlab CI to be still listed as the required workflow to allow PR merging. Which will never happen as it isn't triggered anymore.

In attempt to fix, I removed Gitlab CI from required workflow and replaced by the 3 building ones of the GH CI.

For the record, here is where this setting is located (in the settings of the FreeCAD/FreeCAD repo) :
![image](https://github.com/FreeCAD/FreeCAD/assets/48731257/3aaab773-556c-47b8-bf5f-93a8515916b6)

**Comment by 0penBrain** [comment]
How. Good it seems to have been retroactively applied to pending PR. :) Checkmarks are updated.

**Comment by chennes** [comment]
Yep, sorry, that was me, doing only half the job...

**Comment by sliptonic** [comment]
Learning my way around Github discussions.   Is this topic now resolved?  Do we 'close' it ?

**Comment by chennes** [comment]
I think in much the same way we encourage authors on the Forums to mark things as "[Resolved]" when their question is answered, if the discussion needs no further input, then closing it is the right way to go.

**Comment by 0penBrain** [comment]
Sounds a good rule. :smile: Let's try it.

**Comment by adrianinsaval** [comment]
I still get the branch protection warning:
![image](https://github.com/FreeCAD/FreeCAD/assets/36372335/eb1dc406-f933-4911-8074-5a538c2dbff5)
maybe there's some other setting where it has to be removed? Or does it say that because the PR branch is behind master?

**Comment by chennes** [comment]
The merge is being prevented by being out-of-date, in that case. Rebase to master and force push.

**Comment by adrianinsaval** [comment]
doesn't "rebase and merge" do the same?

**Comment by chennes** [comment]
Yes. But GitHub can't know you plan on doing that, and the branch *is* out of date. So you are bypassing a branch protection.
